app/assets/javascripts/text_components.js

```diff
@@ -0,0 +1,146 @@
+/*
+ * Place all the behaviors and hooks related to the matching controller here.
+ * All this logic will automatically be available in application.js.
+ */
+
```

Comment:
I actually didn’t really add that comment. I changed the file to be vanilla js instead of coffeescript and had to change the comments syntax. Or am I misunderstanding your question?